migration/sql-files/032-add-space-id-wilt.sql

```diff
@@ -1,5 +1,11 @@
 -- Alter the table work_item_link_types
-ALTER TABLE work_item_link_types ADD space_id uuid;
+INSERT INTO spaces (created_at, updated_at, id, name, description) VALUES (now(),now(),'2e0698d8-753e-4cef-bb7c-f027634824a2', 'system.space', 'Initial space');
```

Comment:
@hectorj2f we have designed the migration to be usable from just an SQL file or some Go code. In fact the SQL file parsing is done using bindata and then reading the file back in and executing it.

I've created a PR from an idea I just had: #886 It might need some more love, though.